.github/workflows/node.yml

```diff
@@ -79,7 +81,7 @@ jobs:
                   engine-version: ${{ matrix.engine.version }}
 
             - name: test
-              run: npm test
+              run: npm test -- --testPathIgnorePatterns="ServerModules"
```

Comment:
update the task in package.json
and create a new one for running servermodules test, for example
```
        "test": "npm run build-test-utils && jest --verbose --runInBand --testPathIgnorePatterns='ServerModules'",
        "test-modules": "npm run build-test-utils && jest --verbose --runInBand --testNamePattern='ServerModules'",
```